.github/workflows/release-latest.yml

```diff
@@ -28,6 +28,22 @@ jobs:
       - run: |
           git tag v${{ fromJson(steps.pkg.outputs.json).version }}
           git push origin v${{ fromJson(steps.pkg.outputs.json).version }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@dc7b9719a96d48369863986a06765841d7ea23f6 # 2.0.0
+        with:
+          install: true
+          config-inline: |
+            [worker.oci]
+              max-parallelism = 1
+      - name: Build injection image and push to github packages
+        run: |
+          node scripts/set-version-for-init-image.js ${{ fromJson(steps.pkg.outputs.json).version }}
+          npm pack dd-trace@${{ fromJson(steps.pkg.outputs.json).version }}
```

Comment:
Should this have been done in the npm publish process and stored as an artifact?

Reply:
When referencing a package and version, `npm pack` will download the tarball without installing it. (or pull it from cache). That's what we're going for here.